blockchain/blockchain.go

```diff
@@ -1356,7 +1356,14 @@ func (b *Blockchain) Close() error {
 
 // CalculateBaseFee calculates the basefee of the header.
 func (b *Blockchain) CalculateBaseFee(parent *types.Header) uint64 {
+	// Return zero base fee is a london hardfork is not enabled
 	if !b.config.Params.Forks.IsActive(chain.London, parent.Number) {
+		return 0
+	}
+
+	// Check if this is the first London hardfork block.
+	// Should return chain.GenesisBaseFee ins this case.
+	if parent.BaseFee == 0 {
```

Comment:
This will never be 0, check this part of code: https://github.com/0xPolygon/polygon-edge/blob/db29c8a475c1f62a33213dbef23adee6e76e721c/chain/chain.go#L99

Reply:
Good catch